src/libs/md5.js

```diff
@@ -0,0 +1,219 @@
+/**
+ * Author: Joseph Myers
+ * Accessed from: http://www.myersdaily.org/joseph/javascript/md5.js
+ *
+ * Modified by: Owen Leong (@owenl131)
+ * Date: 15 Oct 2020
+ */

```

Comment:
Does it have a license? If yes, we should add `@license` and add the type of license.

Reply:
Joseph Myers doesn't specify a particular license for his work. He has mentioned in correspondences with other contributors that he places his work in the public domain.

Reply:
Alright.